datadog/resource_datadog_synthetics_test_.go

```diff
@@ -772,6 +772,11 @@ func syntheticsConfigVariable() *schema.Schema {
 					Required:         true,
```

Comment:
Can you quickly add the new `global` allowed enum value in the description for type while we're here ?